src/Generic.jl

```diff
@@ -344,18 +344,17 @@ include("generic/Misc/Rings.jl")
 include("generic/Misc/Localization.jl")
 
 # TODO/FIXME: deprecate aliases, remove in the future
-import ..AbstractAlgebra: @alias
-@alias ResF ResidueFieldElem
-@alias ResField ResidueField
-@alias Res ResidueRingElem
-@alias ResRing ResidueRing
-
-@alias Rat RationalFunctionFieldElem
-
-@alias AbsSeriesRing AbsPowerSeriesRing
-@alias AbsSeriesElem AbsPowerSeriesRingElem
-@alias RelSeriesRing RelPowerSeriesRing
-@alias RelSeriesElem RelPowerSeriesRingElem
+Base.@deprecate_binding ResF ResidueFieldElem
+Base.@deprecate_binding ResField ResidueField
+Base.@deprecate_binding Res ResidueRingElem
+Base.@deprecate_binding ResRing ResidueRing
+
+Base.@deprecate_binding Rat RationalFunctionFieldElem
+
+Base.@deprecate_binding AbsSeriesRing AbsPowerSeriesRing
+Base.@deprecate_binding AbsSeriesElem AbsPowerSeriesRingElem
+Base.@deprecate_binding RelSeriesRing RelPowerSeriesRing
+Base.@deprecate_binding RelSeriesElem RelPowerSeriesRingElem
 
 # Deprecated in 0.34.*
 @alias Frac FracFieldElem
```

Comment:
another new `@alias` I did not touch -- but `Frac` is used a lot in Oscar.jl, so maybe it's good to take this one slower, as there may be in-progress PRs in Oscar affected by this renaming...